modules/conversion-center/assets/js/handlers/contact-buttons.js

```diff
@@ -0,0 +1,33 @@
+export default class ContactButtonsHandler extends elementorModules.frontend.handlers.Base {
+	getDefaultSettings() {
+		return {
+			selectors: {
+				content: '.e-contact-buttons__content',
+				chatButton: '.e-contact-buttons__chat-button',
+				closeButton: '.e-contact-buttons__close-button',
+			},
+		};
+	}
+
+	getDefaultElements() {
+		const selectors = this.getSettings( 'selectors' );
+		return {
+			$content: this.$element.find( selectors.content ),
+			$chatButton: this.$element.find( selectors.chatButton ),
+			$closeButton: this.$element.find( selectors.closeButton ),
```

Comment:
We try to use native JS these days:

`content: this.$element[ 0 ].querySelector( selectors. content ),`

And:

`this.elements.content.classList.toggle( 'is-visible' );`

And:

`this.elements.closeButton.addEventListener( 'click', this. onChatButtonClick.bind( this ) );`

You can decide if you want to re-write this (while it's small), or just keep it in mind for future handlers.

(We want to get rid of jQuery someday, or at least defer the loading of jQuery, so it helps if as little as possible widgets depend on it).

I will leave it up to you to decide :)

Reply:
I am absolutely in favour of using Vanilla JS, so I'm removing jQuery now.
I did see jQuery used in all the other handlers and I did the same, but yessss let's remove it!!
Thank you!

Reply:
Done